xrpl/wallet/main.py

```diff
@@ -17,17 +17,12 @@ class Wallet:
     """
 
     @property
-    def classic_address(self: Wallet) -> str:
+    def address(self: Wallet) -> str:
```

Comment:
Is there any reason not to keep the old one as an Alias?

Reply:
Given the docs still explicitly call our a difference on xrpl.org, I think it can be more confusing to remove this altogether as opposed to showing `address` as the default and providing docs inline in the functions

Reply:
Fair enough - re-added :) (Although I think it's worthwhile to still change everything to use `address` by default)